src/services/database/models/recipe.js

```diff
@@ -23,83 +45,74 @@ const RecipeSchema = new Schema(
     description: {
       type: String,
       trim: true,
+      maxLength: 1000,
     },
     instructions: {
       type: [String],
       required: true,
     },
-    ingredients: [
-      {
-        ingredient: {
-          type: Ingredient.schema, // Assuming Ingredient is a Mongoose model
-          required: true,
-        },
-        amount: {
-          type: String,
-          required: true,
-          trim: true,
-        },
-        unit: {
-          type: String,
-          required: true,
-          trim: true,
-        },
+    ingredients: {
+      type: [RecipeIngredientSchema],
+      validate: {
+        validator: (v) => Array.isArray(v) && v.length > 0,
+        message: 'A recipe must have at least one ingredient.',
       },
-    ],
+    },
     isPublic: {
       type: Boolean,
       default: false,
     },
-    dateCreated: {
-      type: Date,
-      default: Date.now,
-    },
-    dateUpdated: {
-      type: Date,
-      default: Date.now,
-    },
     type: {
       type: String,
       trim: true,
     },
     mealTime: {
       type: String,
       trim: true,
+      enum: ['breakfast', 'lunch', 'dinner', 'snack', 'dessert'],
     },
     cuisine: {
       type: String,
       trim: true,
+      enum: ['italian', 'chinese', 'mexican', 'indian', 'american', 'other'],
     },
     servings: {
       type: Number,
       required: true,
     },
     prepTime: {
-      type: Number, // in minutes
+      type: Number,
       required: true,
     },
     cookTime: {
-      type: Number, // in minutes
-      required: true,
-    },
-    totalTime: {
-      type: Number, // in minutes
+      type: Number,
```

Comment:
You need a cook time, prep time and total time, it could take 5 minutes to cook but 30 minutes to prepare